### PR TITLE
Remove tallclair from some OWNERS files

### PR DESCRIPTION
**Pull request by tallclair**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Remove myself from some owners files. These are mostly areas that I don't feel I have enough expertise to give a good review. I'm trying to cut down on my review noise, to better focus on the areas I have more expertise in.

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

/sig node

### CODE REVIEW

**Comment by cheftako** [comment]
/lgtm

Thanks for the help and work Tim.

**Review by janetkuo — APPROVED** [review]

**Comment by janetkuo** [comment]
/approve 

LGTM for SIG apps

File: pkg/kubelet/server/OWNERS [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+- tallclair
```
Comment (liggitt) [inline pkg/kubelet/server/OWNERS:4]: anyone else from pkg/kubelet/OWNERS approvers that should be hoisted here? I think this means you'll be targeted specifically before bubbling up

File: pkg/kubelet/prober/OWNERS [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+- tallclair
```
Comment (liggitt) [inline pkg/kubelet/prober/OWNERS:4]: anyone else from pkg/kubelet/OWNERS approvers that should be hoisted here? I think this means you'll be targeted specifically before bubbling up

**Comment by liggitt** [comment]
two questions about the sub package OWNERS files, lgtm otherwise

File: pkg/kubelet/server/OWNERS (hunk 1 above)
Comment (tallclair) [reply pkg/kubelet/server/OWNERS:4]: Good to know.

@cheftako I'm nominating you to be the second reviewer for the kubelet server. LMK if you don't want it, or want to nominate anyone else.

File: pkg/kubelet/prober/OWNERS (hunk 2 above)
Comment (tallclair) [reply pkg/kubelet/prober/OWNERS:4]: I added sig-node-approvers here, do you know if that works? Will it weight me and everyone in sig-node-approvers equally?

Comment (liggitt) [reply pkg/kubelet/prober/OWNERS:4]: I'm not sure... paging @kubernetes/sig-contributor-experience-pr-reviews in case they know

Comment (cblecker) [reply pkg/kubelet/prober/OWNERS:4]: yeah, all the folks including from the alias will have equal weight

**Pull request merged**